src/Tasks/Microsoft.Common.props

```diff
@@ -174,4 +174,20 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <NuGetPropsFile Condition="'$(NuGetPropsFile)'==''">$(MSBuildToolsPath)\NuGet.props</NuGetPropsFile>
   </PropertyGroup>
   <Import Condition="Exists('$(NuGetPropsFile)')" Project="$(NuGetPropsFile)" />
+
+  <PropertyGroup>
```

Comment:
Where are these used?

Reply:
In TaskExecutionHost.cs line 1240. This is where we turn off logging for individual item types. This is the actual payload of the change, listing items and item metadata that we don't care about.